src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const adder = (num) => {
+    if(num === undefined) {
+      const currentTotal = total;
+      total = 0;
```

Comment:
The resetting of the `total` variable is now correctly implemented. However, ensure that the function name and file name align with the task requirements.